packages/@aws-cdk/aws-lambda/lib/private/sort.ts

```diff
@@ -0,0 +1,17 @@
+export function sortedStringify(value: any, space?: string | number): string {
+  return JSON.stringify(value, (_, v) => sortKeys(v), space);
+}
+
+function sortKeys(obj: any) {
+  if (typeof(obj) !== 'object' || Array.isArray(obj)) {
+    return obj;
+  }
+  let newObject = {};
+  Object.keys(obj).sort().forEach(k => {
+    newObject = {
+      ...newObject,
+      [k]: obj[k],
+    };
```

Comment:
```suggestion
    newObject[k] = obj[k];
```

Reply:
Doesn't work. I get the error

```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{}'.
  No index signature with a parameter of type 'string' was found on type '{}'
```

Also, this is written in a way that the key is inserted *after* the existing `newObject`.